desk/ted/vitals/connection-check.hoon

```diff
@@ -33,7 +33,13 @@
   ::  set pending to %trying-local
   ;<  ~  bind:m  (update-status [%trying-local ~])
   ::  check if we can contact our own galaxy
-  ;<  gqos=qos:ames  bind:m  (scry:io qos:ames ~[%gx %vitals %galaxy %vitals-qos])
+  ;<    gqos=qos:ames
+      bind:m
+    =/  mm  (strand ,qos:ames)
+    ^-  form:mm
+    ?:  ?=(%czar (clan:title our))
+      (pure:mm [%live *@da])
```

Comment:
> skip sponsor checks for galaxies (avoids a crash in /app/vitals/hoon)

Yeah I wonder if it might be better to just update the `/sponsor` and `/galaxy` scry endpoints in the agent to, instead of asserting that we're not a galaxy, just always return the most-positive `%vitals-qos` if we are a galaxy. As a galaxy, we are our own sponsor, and we are ourselves, so we know we are live, and were last reached at the present moment (`[%live now.bowl]`).

That way, we don't end up with checks like this one in every single place that wants to scry into vitals.

Reply:
That would be my preference. I have a bit of "Chesterton's Fence" anxiety about removing the assertions, but my feeling is that we can try it and keep it if nothing breaks.

Reply:
I personally wouldn't worry about it in this case. But indeed good intuition to be careful/stay vigilant! (: